tools/run_tests/dockerize/docker_run.sh

```diff
@@ -32,13 +35,39 @@ else
   cp -r "$EXTERNAL_GIT_ROOT/$RELATIVE_COPY_PATH"/* "/var/local/git/grpc/$RELATIVE_COPY_PATH"
 fi
 
+# ensure the "reports" directory exists
+mkdir -p reports
+
+# TODO(jtattermusch): this is garbage, remove it.
 $POST_GIT_STEP
 
-if [ -x "$(command -v rvm)" ]
+# TODO(jtattermusch): is this necessary?
+cd /var/local/git/grpc
```

Comment:
Less coupling between 2 scripts is nice I guess.

Reply:
Will look into this in a followup PR.